src/modules/job-manager/Makefile.am

```diff
@@ -112,7 +112,7 @@ test_ldadd = \
 	$(top_builddir)/src/common/libjob/libjob.la \
 	$(top_builddir)/src/common/libflux-internal.la \
 	$(top_builddir)/src/common/libflux-core.la \
-	$(LIBPTHREAD) $(JANSSON_LIBS)
```

Comment:
Same here, $(JANSSON_LIBS) Is missing from LIBADD.

Also, what's the deal with `$(fluxmod_libadd)`?  All the modules add it, but it doesn't seem to be defined anywhere?  (possibly I'm missing something).

Reply:
> Also, what's the deal with $(fluxmod_libadd)? All the modules add it, but it doesn't seem to be defined anywhere? (possibly I'm missing something).

I'm going to take a guess.  Perhaps `fluxmod_ldflags` was slowly copied around and cutters/pasters assumed `fluxmod_libadd` or `fluxmod_ldadd` was also defined?  Apparently this issue has existed as far back as `0.1.0`,  I'll do a bit of cleanup with this too.